rules/cloud/aws/aws_ec2_vm_export_failure.yml

```diff
@@ -16,10 +17,7 @@ detection:
     errorMessage: '*'
   filter2:
     errorCode: '*'
-  filter3:
-    eventName: 'ConsoleLogin'
```

Comment:
I don't think this needs `ConsoleLogin` tho.

Reply:
Filter3 means than you can have a CreateInstanceExportTask error log whitout `errorMessage` or `errorCode` .
BUT you need a log with eventName `ConsoleLogin` AND `CreateInstanceExportTask` in the same time 🙄

Reply:
So - should we remove `filter1` and `filter2`, keep `filter3` but remove `eventName: 'ConsoleLogin'`, since we can't have two eventNames for one event?

Reply:
I'm just curious, because in elastic I've  gotten alerted with just:

```
event.dataset:aws.cloudtrail and event.provider:ec2.amazonaws.com and event.action:CreateInstanceExportTask and 
event.outcome:failure
```

Reply:
Added `filter3` back, without `eventName: 'ConsoleLogin'` - I think that should work.

Reply:
event.outcome is a ECS build from other fields 
```yaml
detection:
  selection_event:           
    eventName: 'CreateInstanceExportTask' #event.action
    eventSource: 'ec2.amazonaws.com' #event.provider
  selection_fail_info:  #event.outcome
     - errorMessage: '*'
     - errorCode: '*'
     - responseElements|contains: 'Failure'
 condition: all of them
```
remove the # like just add to info for @austinsonger

Reply:
If we expect all the fail info to apply, would this be sufficient?
```
  selection:
    eventName: 'CreateInstanceExportTask'
    eventSource: 'ec2.amazonaws.com'
    errorMessage: '*'
    errorCode: '*'
    responseElements|contains: 'Failure'
  condition: selection
  ```

Reply:
it is `AND for all` so you MUST have errorMessage + errorCode +responseElements.
Should keep the OR in first time and tune is they are FP.

Reply:
HI , for me the rule is a good step to start (We can always refine it later based on feedback).
So if you agree, let's resolve the dicution and I will merge the rule.

Reply:
Sounds good, thanks